index.bs

```diff
@@ -236,25 +237,25 @@ aligned (8) class AV1CodecConfigurationRecordv1 {
   unsigned int (1) marker = 1;
   unsigned int (3) version = 1;
   unsigned int (3) seq_profile;
+  unsigned int (1) still_picture;
+  unsigned int (5) seq_level_idx_0;
+  unsigned int (1) seq_tier_0;
   unsigned int (3) reserved = 0;
   unsigned int (1) initial_presentation_delay_present;
   if (initial_presentation_delay_present) {
     unsigned int (4) initial_presentation_delay_minus_one;
   } else {
     unsigned int (4) reserved = 0;
   }
-  unsigned int (1) sequence_parameters_present;
-  if (sequence_parameters_present) {
-    unsigned int (1) still_picture;
-    unsigned int (5) seq_level_idx_0;
-    unsigned int (1) seq_tier_0;
-    unsigned int (4) bit_depth;
+  unsigned int (1) color_parameters_present;
+  if (color_parameters_present) {
     unsigned int (1) monochrome;
+    unsigned int (4) bit_depth;
```

Comment:
You could replace this field with "high_bitdepth" and "twelve_bit", each 1 bit as per the spec, in case the two saved bits help aligning stuff in any way (The entire color_parameters struct + the present bit could be all in the same byte that way, for example).

Monochrome goes right after those two in the spec, so you'd also be able to keep them in order.